paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1095,7 +1095,11 @@ def on_enter_deployed(self) -> None:
         line = f"Deployment of {self.commit[:8]} for {self.deploy_group} complete"
         _log(service=self.service, component="deploy", line=line, level="event")
         self.send_manual_rollback_instructions()
-        if not (self.any_slo_failing() and self.auto_rollbacks_enabled()):
+        if self.any_slo_failing() and self.auto_rollbacks_enabled():
+            self.ping_authors(
+                "Because an SLO is currently failing, we will not automatically certify. Instead, we will wait indefinitely until you click one of the buttons above."
```

Comment:
is this actually indefinitely or do we have a timeout elsewhere that would kill this process? this might be the releng in me, but it seems a little wasteful to hold onto a CI executor forever (well, really until something in jenkins goes sideways or the cluster scales down and that agent gets caught in the fray and is terminated)

that said, if this is actually gonna wait forever, it's probably fine to ship this first and add a timeout later if adding a timeout is non-trivial here

Reply:
There's a timeout (3 hours default but overridable in deploy.yaml). I guess I could mention it here.

Reply:
ah cool - I wasn't sure if that timeout applied here or not. 

up to you if you want to mention it!